gnmawsgr/templates/n.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+        "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+    <font size="+1">
+        Item restore not started because a restore job for that item is already underway.
```

Comment:
please can we change the filenames to something a little more descriptive? thanks